contracts/PathsDB.sol

```diff
@@ -26,4 +26,8 @@ contract PathsDB {
 
       return checkpoints;
     }
+
+    function getPathLength() constant returns (uint) {
```

Comment:
I needed to create this method because it was the only way I could figure out how to get the length for my getPath function in the paths controller.